src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const photoLinks = document.querySelectorAll('.list-item__link');
+const mainPhoto = document.getElementById('largeImg');
+
+[...photoLinks].forEach(link =>
+  link.addEventListener('click', (e) => {
+    e.preventDefault();
+    mainPhoto.src = e.currentTarget.href;
+  })
+);
```

Comment:
As description for this task says:

> Notes:
> 
> Use event delegation.

So you should use event delegation for UL instead of creating an event listener for every LI